### PR TITLE
{lib}[GCCcore/10.3.0] libidn v1.36

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/54030135c4e451f38df12eb8b23cec7b for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.2.2004, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/9f5d1d6eba7a5a3c4fb663b4429ea818 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3530.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/64e2b37e57accaea94abfa27db1014e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2640.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/b902ea14f27a40fc31c660c07148e230 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7d45741341bf438a5226999bd3db8094 for a full test report.

File: easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'libidn'
+version = '1.36'
```
Comment (branfosj) [inline easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb:4]: See #12663 - it looks like we should use a 2.x version?

Comment (Micket) [reply easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb:4]: Hm yes. I wonder if we should introduce a new `libidn2` for it instead? https://www.gnu.org/software/libidn/#libidn2

Comment (branfosj) [reply easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb:4]: Is that needed? Other than `wget` I cannot see `libidn` used as a dependency in recent easyconfigs?

Comment (Micket) [reply easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb:4]: Well.. it just seems neater; they are updated independently https://ftp.gnu.org/gnu/libidn/ (in fact, 1.36 is newer than 2.3.0). The library is called libidn2, the binary is called idn2. I see debian and centos are packaging libidn2 separately (and both might be required, my centos machine has both at least)

Comment (Micket) [reply easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb:4]: I really think something like this #12665 is most appropriate

Comment (branfosj) [reply easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb:4]: OK. That makes sense.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/11689763fde1589f79cb75df44a370c3 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**